datafusion/proto/Cargo.toml

```diff
@@ -51,6 +51,7 @@ pbjson = { version = "0.6.0", optional = true }
 prost = "0.12.0"
 serde = { version = "1.0", optional = true }
 serde_json = { workspace = true, optional = true }
+strum = { version = "0.25.0", features = ["derive"] }
```

Comment:
BTW since this module is only used in the tests, we should put it in the `dev-dependencies` section, so everyone using `datafusion-proto` does not need to compile it as well

I did this in 0a41737b1